### PR TITLE
review this branch

### PR DESCRIPTION
**Pull request by MelMott**
messed this really up review please

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,6 +1,2 @@
 def twoSum(n, t):
-    def twoSum(n, t, 4):
-    for i in range(len(n)):
-        for j in range(i + 1, len(n)):
-            if n[j] == t - n[i]:
-                return [i, j]
+   pass
```
Comment (rmora101) [inline functions/two_sum.py]: dont think this pass is necessary

File: functions/two_sum.py [hunk 2]
```diff
@@ -1,6 +1,2 @@
 def twoSum(n, t):
```
Comment (rmora101) [inline functions/two_sum.py:4]: theres two def functions

File: functions/two_sum.py (hunk 1 above)
Comment (bellaiam) [inline functions/two_sum.py]: still have pass in your code, please take it

Comment (avillal66) [inline functions/two_sum.py]: Please remove pass

File: tests/two_sum_test.py [hunk 3]
```diff
@@ -1,41 +1,2 @@
 import pytest
 from functions.two_sum import twoSum
-
-import pytest
```
Comment (bellaiam) [inline tests/two_sum_test.py:4]: dry your code, you have duplicated import statements

**Review by avillal66 — CHANGES_REQUESTED** [review]

File: tests/two_sum_test.py [hunk 4]
```diff
@@ -1,41 +1,2 @@
...
-    # Arrange
-    n = [3, 3]
-    t = 6
-    j = 4 
-
-    # Act
-    answer = twoSum(n, t)
-
-    # Assert
-    assert answer == [0, 1]
```
Comment (bellaiam) [inline tests/two_sum_test.py]: what is the minus sign doing here?

File: functions/two_sum.py [hunk 5]
```diff
@@ -1,6 +1,5 @@
...
-    def twoSum(n, t, 4):
     for i in range(len(n)):
         for j in range(i + 1, len(n)):
             if n[j] == t - n[i]:
-                return [i, j]
```
Comment (bellaiam) [inline functions/two_sum.py:6]: you have two return statements

**Review by bellaiam — APPROVED** [review]

**Review by rmora101 — COMMENTED** [review]
LOOKS gud does it work tho?

**Review by avillal66 — APPROVED** [review]
Great Job!

**Review by rmora101 — APPROVED** [review]

**Pull request merged**